SMatrixADT.c

```diff
@@ -72,12 +72,12 @@ int main ()
                 printf ("Matrix SUBTRACTION error!\n");
             break;
         case TRANSPOSE:
-            if (SMTX_transpose (&A, &B, &C)!= SMTX_ERROR)
-                SMTX_print (&C);
+            if (SMTX_transpose (&A, &B)!= SMTX_ERROR)
```

Comment:
There should be only two parameters for SMTX_transpose function instead of three